src/EventStore.Core/Util/Opts.cs

```diff
@@ -218,6 +218,9 @@ public static class Opts {
 
 		public const string CertificateSubjectNameDescr = "The certificate subject name.";
```

Comment:
@pgermishuys To avoid any confusion with the certificate loading from store, can we consider renaming:
- --certificate-subject-name to --certificate-store-subject-name? 
- --certificate-thumbprint to --certificate-store-thumbprint?

what do you think?

Reply:
sorry, forget this, I misread the other option as `CertificateCommonName`

Reply:
Renaming the existing options be a breaking change. The best we can do is to describe the new option in a non confusing way.